docs/observability/README.md

```diff
@@ -0,0 +1,104 @@
+
+```mermaid
+graph
+A[Application] --> B((Logs))
+A --> C((Metrics))
+A --> D((Traces))
+```
+
+## Metrics
+
+Metrics are a way to measure the state of your application. Metrics are usually numerical values that can be aggregated and visualized. Metrics are often used to create alerts and dashboards.
+
+We use the [OpenMetrics][openmetrics] format for metrics. This is a text-based format that is easy to parse and understand. It is also the format used by Prometheus, which is the most popular metrics system.
+
+[openmetrics]: https://openmetrics.io/
+
+[:octicons-arrow-right-24: Get started with metrics](./metrics.md)
+
+### Prometheus
+
+[Prometheus][prometheus] is a time-series database that is used to store metrics. It is a very powerful tool that can be used to create alerts and dashboards. Prometheus is used by many open source projects and is the de facto standard for metrics in the cloud native world.
+
+Prometheus is a pull-based system. This means that Prometheus will scrape (pull) metrics from your application. This is in contrast to a push-based system, where your application would push metrics to a central system.
+
+[prometheus]: https://prometheus.io/
+
+```mermaid
+graph LR
+  Grafana --> Prometheus
+  Prometheus --GET /metrics--> Application
+```
+
+### Grafana
+
+[Grafana][grafana] is a tool for visualizing metrics. It is used to create dashboards that can be used to monitor your application. Grafana is used by many open source projects and is the de facto standard for metrics in the cloud native world.
+
+[:octicons-arrow-right-24: Access Grafana here][nais-grafana]
+
+[grafana]: https://grafana.com/
+[nais-grafana]: https://grafana.nais.io/
+
+## Logs
+
+Logs are a way to understand what is happening in your application. They are usually text-based and are often used for debugging. Since the format of logs is usually not standardized, it can be difficult to query and aggregate logs and thus we recommend using metrics for dashboards and alerting.
+
+Logs are collected automatically by [fluentd][fluentd] and stored in [Elasticsearch][elasticsearch] and accessible via [Kibana][kibana].
+
+[fluentd]: https://www.fluentd.org/
+[elasticsearch]: https://www.elastic.co/elasticsearch/
+[kibana]: https://www.elastic.co/kibana/
+
+```mermaid
+graph LR
+  Application --stdout/stderr--> Fluentbit
+  Fluentbit --> Elasticsearch
+  Elasticsearch --> Kibana
+```
+
+[:octicons-arrow-right-24: Configure your logs](./logs/README.md)
+
+## Alerts
+
+Alerts are a way to notify you when something is wrong with your application. Alerts are usually triggered when a metric or log entry matches a certain condition. Alerts are often used to notify you when something is wrong with your application.
+
+Alerts are based on application metrics and use [Prometheus Alertmanager][alertmanager] to send notifications to Slack. Alerts are configured by each team for their applications using the [`alert`](./alerts/) resource.
```

Comment:
```suggestion
Alerts are a way to notify you when something is wrong with your application, and are usually triggered when a metric or log entry matches a certain condition.

Alerts in NAIS are based on application metrics and use [Prometheus Alertmanager][alertmanager] to send notifications to Slack. 

The [`alert`](./alerts/) resource can be used to configure alerts for your applications.
```

Trying to avoid starting all the sentences with "Alerts are" 😄